### PR TITLE
improving test coverage

### PR DESCRIPTION
**Pull request by amandadumi**
expanding tests for previous misses (mistakes in input, large molecules for torsions). Molecule class raises KeyError instead of printing error for unaccepted atom symbol.

### CODE REVIEW

**Comment by amandadumi** [comment]
I have formatted the large arrays in the tests according to pep8. It isn't the prettiest, so if there is something you'd rather me do, let me know!

**Comment by dlf57** [comment]
Would it be fine if we broke pep8 for the arrays? Only because it makes it way easier for sanity checks to see if things match up.

**Comment by amandadumi** [comment]
I manually formatted these so a line of the array does not exceed 80 characters. Does this seem acceptable?

File: chemreps/utils/molecule.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def sym2num(self, sym):
             atomic_number = self.__nuc['{}'.format(sym)]
             return atomic_number
         except:
-            print('{} is not defined.'.format(sym))
+            raise KeyError('{} is not defined.'.format(sym))
```
Comment (shivupa) [inline chemreps/utils/molecule.py]: Nice!

**Comment by shivupa** [comment]
Yeah I think that looks nice and the changes look good

File: tests/bat_test.py [hunk 2]
```diff
@@ -1,63 +1,298 @@
...
     rep = bat('data/sdf/butane.sdf', bagger.bags, bagger.bag_sizes)
     assert np.all(np.abs(bat_true-rep) <= 1e-4) == True
 
+    with pt.raises(NotImplementedError):
+        rep = bat('data/xyz/butane.xyz', bagger.bags, bagger.bag_sizes)
+
+    with pt.raises(NotImplementedError):
+        bagger = BagMaker('BAT', 'data/xyz/')
+
+    pen_bat_true = np.array([36.8438, 36.8438, 36.8438, 36.8438, 36.8438,
```
Comment (dlf57) [inline tests/bat_test.py]: I feel like if we add this in we can remove the butane.sdf test as this tests the same thing as that and more by hitting all the torsions

**Comment by dlf57** [comment]
Everything looks good! I just don't know if we want to have the two tests for reps on BAT as the penicillin one should cover it.

**Pull request merged**